src/strife/f_wipe.c

```diff
@@ -88,18 +94,27 @@ wipe_doColorXForm
   int	height,
   int	ticks )
 {
-    byte *cur_screen = wipe_scr;
-    byte *end_screen = wipe_scr_end;
+    pixel_t *cur_screen = wipe_scr;
+    pixel_t *end_screen = wipe_scr_end;
     int   pix = width*height;
     int   i;
     boolean changed = false;
 
+    // [crispy] reduce fail-safe crossfade counter tics
+    fade_counter--;
+
     for(i = pix; i > 0; i--)
     {
-        if(*cur_screen != *end_screen)
+        if(*cur_screen != *end_screen && fade_counter)
         {
             changed = true;
+#ifndef CRISPY_TRUECOLOR
             *cur_screen = xlatab[(*cur_screen << 8) + *end_screen];
+#else
+            // [crispy] perform crossfading effect with 13 given opacity steps, multipled by 19:
+            // 247, 228, 209, 190, 171, 152, 133, 114, 95, 76, 57, 38, 19
+            *cur_screen = I_BlendOver(*end_screen, *cur_screen, fade_counter * 19);
```

Comment:
Well, you could write the `I_BlendOver()` result into a temporary variable of type `pixel_t` and check if its value is the same as `*cur_screen`. They should be in the same format at this point.

Reply:
But what for exactly? To get rid of `fade_counter`, or...?

Reply:
Yep, and of the bug you described two days ago.

Reply:
Got the idea, but I have to understand how to do it properly. Current crossfading is okay, but knowing abilities of TrueColor... It definitely can be better. The only thing under the question is - what `amount` of alpha should `I_BlendOver` receive every call while crossfading then? Is to 50%, i.e. 127?

Reply:
I see. Fading in palette mode is cummulative whereas in Truecolor mode it's absolute. 🤔

Reply:
I'm afraid I'm going to give up. Matters no how or where I'm trying to reduce increased value of `fade_counter` for smooth crossfading, it's always longer in geometric progression than actual effect. I can't understand neither why nor how to avoid it, it's like going around circles. 🙁

Reply:
I think it's okay, thank you!

Reply:
Sigh. Thank you, at least for now we can go "vanilla way", knowing the fact that it can potentially go "epic way". I definitely want to get back to it eventually, but with fresh look.

Reply:
Small abstract idea. I need to try something different. Not a multiplier, but strict array, like...
* `static const int fade_steps[] = {240, 230, 220, ...};`
called by 
* `*cur_screen = I_BlendOver(*end_screen, *cur_screen, fade_steps[fade_counter]);`

I have a suspicion that delays were happened because crossfade was finished, but `fade_counter` was still ticking.